oemer/classifier.py

```diff
@@ -12,6 +12,7 @@
 from sklearn.ensemble import AdaBoostClassifier, RandomForestClassifier, GradientBoostingClassifier, BaggingClassifier
 from sklearn.linear_model import RidgeClassifier
 from sklearn.model_selection import GridSearchCV
+import tensorflow as tf
```

Comment:
The `import ...` should be before `from ... import ...`.

Reply:
This is the same style as in all other places to alias tensorflow. E.g. unet.py or inference.py. Also it matches the tensorflow documentation e.g. at https://www.tensorflow.org/tutorials/quickstart/beginner